bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,48 @@
+# speedtest Binding
```

Comment:
```suggestion
# Speedtest Binding
```
It looks to be a trademark and the correct spelling is with a capital and no space.

Reply:
done